lib/src/base-styles/_nx-forms.scss

```diff
@@ -81,3 +81,20 @@
   margin: 0 0 $nx-spacing-xl 0;
   padding: 0;
 }
+
+// Inline form group
+.nx-form-row--inline-form {
```

Comment:
Could flex just be applied to `.nx-form-row` and the modifier class be avoided.  I tried this with the other row example and it didn't hurt, I think it actually improved it slightly by removing a height discrepancy between the form groups and the row.  If you're worried about wrapping you could always add `flex-wrap`

Reply:
> Could flex just be applied to `.nx-form-row` and the modifier class be avoided. I tried this with the other row example and it didn't hurt, I think it actually improved it slightly by removing a height discrepancy between the form groups and the row. If you're worried about wrapping you could always add `flex-wrap`

It can be and it's a much more elegant solution because it doesn't require the use of any custom classes when it's used with `nx-btn-bar`: https://github.com/sonatype/sonatype-react-shared-components/pull/164/commits/768f86743bae3a6b994fe0f05ae2f3ea359b6d52